### DISCUSSION TITLE
输入!#标签时，总会在!和#标签之间增加空格，想保持 !#标签，怎么办？

### DISCUSSION THREAD

**Discussion by webrunneronSun** [Yaozhuwa/easy-typing-obsidian#101] [Q&A]
输入代码有时需要用 !#标签 来排除这个标签，但是每次输入后都会变成 ! #标签，! 和# 之间多了一个空格，怎样才能保持 !#标签 呢？

**Comment by Yaozhuwa** [comment]
插件设置中添加正则表达式如下
```
!#[\u4e00-\u9fa5a-zA-Z\/]+|--
```
![image](https://user-images.githubusercontent.com/28787183/204439644-30e411df-4521-40bf-9872-c827d8193d85.png)

**Comment by webrunneronSun** [comment]
谢谢！插件非常好用。谢谢！
